img/style.css

```diff
@@ -10,7 +10,7 @@ HTML {
 BODY {
   font-size: 8pt;
   font-family: Arial, Helvetica, Verdana;
-  background-color: #EBE4D6;
+  background-color: #ebe4d6;
```

Comment:
?!

Reply:
Wrzucam bez style.css.
Lessc ma problem z przyjęciem --no-color u mnie.

Reply:
Pisałem już:
```bash
# lessc --version
lessc 2.7.2 (Less Compiler) [JavaScript]
```
Jaki u Ciebie jest kompilator less i w jakiej wersji?
Możliwe, że już pisałeś i przeoczyłem.

Reply:
Tak to to zaraz zainstaluje najnowsza wersję.

Reply:
Na samym serwerze gdzie działa demo jest:
```bash
# lessc --version
lessc 3.0.4 (Less Compiler) [JavaScript]
```